.github/workflows/checks.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
+      - next-v*
```

Comment:
Should we keep that for `next-v6`, or revert it ?

Reply:
I'm good with keeping this.